### PR TITLE
[REM] core, web: Odoo 16 deprecations

### PR DESCRIPTION
**Pull request by xmo-odoo**
Remove symbols and options deprecated as of Odoo 16.

`odoo.web.controllers.main` is a bit dodgy as all its contents got deprecated but the module itself remained, so technically it's possible that there are dead imports lying around. Remove all the contents but only deprecate the module.

### CODE REVIEW

**Review by Julien00859 — COMMENTED** [review]
My bad for odoo.addons.web.controllers.main, should had thought about the module itself. I'm fine with the extended deprecation notice. Please update the tests at `odoo/addons/base/tests/test_configmanager.py` for `--longpolling_port`

**Review by xmo-odoo — COMMENTED** [review]
@robodoo override=ci/security

File: odoo/addons/base/tests/config/14.0.conf [hunk 1]
```diff
@@ -34,7 +34,6 @@ log_db_level = warning
 log_handler = :INFO
 log_level = info
 logfile = 
-longpolling_port = 8072
```
Comment (Julien00859) [inline odoo/addons/base/tests/config/14.0.conf:37]: The test is about making sure that the default configuration of the oldest supported version is still compatible with master. I suggest we remove this `14.0.conf` file, checkout 16.0 and do `--config odoo/addons/base/tests/config/16.0.conf --save --stop`. The diff should ultimately be
```diff
---odoo/addons/base/tests/config/14.0.conf
+++odoo/addons/base/tests/config/16.0.conf
-longpolling_port = 8072
+gevent_port = 8072
```

**Review by Julien00859 — APPROVED** [review]
top merci :)

**Pull request closed without merging**